### PR TITLE
acronym-list-empty-state Adds condition to check for empty results an…

### PR DESCRIPTION
**Pull request by NFabrizio**
…d no results component

### CODE REVIEW

File: src/AcronymList.js [hunk 1]
```diff
@@ -32,6 +33,12 @@ const acronymCategories = (list = []) => {
 };
 
 const AcronymList = (props) => {
+  if (props.list && !props.list.length) {
+    return (
+      <NoResultsPage />
+    );
+  }
+
```
Comment (tsrahm) [inline src/AcronymList.js]: Should we do the ```props.list``` check for the ```map``` method as well?

Comment (nlkluth) [reply src/AcronymList.js]: Yeah maybe destructure list off with a default assignment

Comment (NFabrizio) [reply src/AcronymList.js]: 👍 Updated.

File: src/AcronymList.js [hunk 2]
```diff
@@ -35,7 +36,19 @@ const isLiked = (likesIds, definitionId) => {
...
 const AcronymList = (props) => {
-  const { list, like, isAuthenticated, likesIds } = props;
+  const {
+    list = [],
+    like,
+    isAuthenticated,
+    likesIds
+  } = props;
+
+  if (!list.length || !list.map) {
```
Comment (nlkluth) [inline src/AcronymList.js]: can a list not have map but have a length?

Comment (tsrahm) [reply src/AcronymList.js]: If list is a map, list.length will be undefined, so this if statement will evaluate to true and return <NoResultsPage /> regardless of the list value.

Comment (NFabrizio) [reply src/AcronymList.js]: Added an update that should handle that.

**Pull request merged**